controller/workitemtype.go

```diff
@@ -95,9 +133,14 @@ func (c *WorkitemtypeController) ListSourceLinkTypes(ctx *app.ListSourceLinkType
 
 // ListTargetLinkTypes runs the list-target-link-types action.
 func (c *WorkitemtypeController) ListTargetLinkTypes(ctx *app.ListTargetLinkTypesWorkitemtypeContext) error {
+	spaceID, err := uuid.FromString(ctx.ID)
+	if err != nil {
+		return errors.NewNotFoundError("spaceID", ctx.ID)
+	}
+
 	return application.Transactional(c.db, func(appl application.Application) error {
 		// Test that work item type exists
-		_, err := appl.WorkItemTypes().Load(ctx.Context, ctx.WitID)
+		_, err := appl.WorkItemTypes().Load(ctx.Context, spaceID, ctx.WitID)
```

Comment:
Not sure we technically need to fetch WIT based on SpaceID and WitID. WitID is designed to be unique on it's own. But it doesn't hurt.